CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.211 Jul 26 2022
+- Added `Add` method to label model.
+
```

Comment:
@tirthct since the release 0.0.210 was not created yet, maybe we can add this change there (no need to create the 0.0.211).